extensions/quartz/deployment/pom.xml

```diff
@@ -1,58 +1,62 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0"
-  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <parent>
-    <artifactId>quarkus-quartz-parent</artifactId>
-    <groupId>io.quarkus</groupId>
-    <version>999-SNAPSHOT</version>
-    <relativePath>../</relativePath>
-  </parent>
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-quartz-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../</relativePath>
+    </parent>
 
-  <modelVersion>4.0.0</modelVersion>
+    <modelVersion>4.0.0</modelVersion>
 
-  <artifactId>quarkus-quartz-deployment</artifactId>
-  <name>Quarkus - Scheduler Quartz - Deployment</name>
+    <artifactId>quarkus-quartz-deployment</artifactId>
+    <name>Quarkus - Scheduler Quartz - Deployment</name>
```

Comment:
Either "Quartz Scheduler" or "Scheduler - Quartz" or just "Quartz" but the current order looks weird.

I suppose if we judge that `quarkus-quartz` is good enough, it should be `Quartz`. Then we can detail in the description and the metadata of the extension.

Reply:
Let's go with `Quartz`

Reply:
Well, "Scheduler" was there for a good reason - it's a link to the Scheduler extension this Quartz extension is build on (and basically extends it's functionality).

Reply:
I can see that but it's hard to grasp IMHO. Maybe something like `Scheduler - Quartz` might be better.

And in this case, why is the extension named `quarkus-quartz` and not `quarkus-scheduler-quartz`? Not saying I want the latter, just asking if we should be consistent somehow.